daemon/src/collab_settlement/maker.rs

```diff
@@ -194,12 +200,11 @@ impl Actor {
                             emit_failed(order_id, source, &executor).await;
                         }
                         Failed::AfterReceiving { source, settlement } => {
-                            // TODO: proceed with the transaction when taker will be able to handle
+                            // TODO: proceed with the transaction when taker will be able to handle that case.
                             tracing::trace!(
                         ?settlement,
                         "Failed after receiving. Ideally, we should be able to act upon this settlement"
                     );
-                            // that case.
                             emit_failed(order_id, source, &executor).await;
```

Comment:
Is it only me or does this indentation feel a bit weird? 😅

Reply:
this is just rustfmt rule - the call to the macro is at the some indent :)
However, I concur - looks pretty weird :)